docs/source/daml/code-snippets/Keys.daml

```diff
@@ -0,0 +1,203 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Do we run this through `daml test` or some other form of interpretation? We should definitely have this checked by a machine to make sure we don't break it ever again.

Reply:
Ok, I just saw we do. Consider this answered.